manifests/a/AMSSoftware/BusinessCardMaker/1.0/AMSSoftware.BusinessCardMaker.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: AMSSoftware.BusinessCardMaker
+PackageVersion: 1.0

```

```suggestion
PackageVersion: 9.15
```
Should Match Control Panel